docs/tables/azure_key_vault_secret.md

```diff
@@ -2,6 +2,8 @@
 
 Azure Key Vault is a cloud service for securely storing and accessing secrets. A secret is anything that you want to tightly control access to, such as API keys, passwords, certificates, or cryptographic keys.
 
+Note: If we intend to list the secrets by vault name using the WHERE clause (select * from azure_key_vault_secret where vault_name = 'test-vault';), the vault name must be in lowercase, even if the vault name appears in camel case in the portal.
```

Comment:
Since we are converting the name to lower case in the code, do we need to add this line to the doc?

Reply:
> Since we are converting the name to lower case in the code, do we need to add this line to the doc?

@misraved, currently, we use the ToLower function to verify the vault name both from the parent hydrate and the Secret API response. However, we are not streamlining the result to lowercase.

In the `azure_key_vault` table, the vault name matches the user-provided name as displayed in the portal. Nevertheless, the response from the Secret API call consistently appears in lowercase.

For those who wish to join data from both the `azure_key_vault` and `azure_key_vault_secret` tables using the vault name sourced from the `azure_key_vault` table, it's important to note that while data can be fetched, it might lead to empty rows. This is due to Steampipe's filtering at the level of casing, meaning that any vault name casing discrepancies will prevent the display of results.

Should any concerns or uncertainties arise, please feel free to reach out. We can sync up. Thank you!